servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcRouter.java

```diff
@@ -302,7 +311,7 @@ public Completable closeAsync() {
                         public Completable closeAsyncGracefully() {
                             return route.closeAsyncGracefully();
                         }
-                    }, executionStrategy == null ? HttpExecutionStrategies.anyStrategy() : executionStrategy),
+                    }, executionStrategy == null ? HttpExecutionStrategies.defaultStrategy() : executionStrategy),
```

Comment:
Is it expected that here we use `HttpExecutionStrategies.defaultStrategy()`, while line 386 uses `GrpcExecutionStrategies.defaultStrategy()`?